src/commands/utils/help.ts

```diff
@@ -45,7 +45,7 @@ export const help: CommandDefinition = {
                     // Slide into the DMs
                     author.createDM()
                         .then(async (dmChannel) => {
-                            const response = await selectorMsg.channel.send(`${msg.author}, I've DM'd you with the list of the commands you can use!`);
+                            const response = await selectorMsg.reply(`${msg.author}, I've DM'd you with the list of the commands you can use!`);
```

Comment:
```suggestion
                            const response = await msg.reply(`${msg.author}, I've DM'd you with the list of the commands you can use!`);
```
Seems better to trigger it to the actual message from the user requesting `help`.